torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2363,6 +2403,23 @@ def _make_tensor_helper(shape, low=None, high=None):
                                 dtypes=[torch.bfloat16])),
                    safe_casts_outputs=True,
                    handles_complex_extremals=False),
+    OpInfo('lerp',
+           dtypes=floating_and_complex_types(),
+           # CUDA and ROCM don't support complex inputs
+           dtypesIfCUDA=floating_types_and(torch.half),
+           dtypesIfROCM=floating_types_and(torch.half),
+           sample_inputs_func=sample_inputs_lerp,
+           skips=(
+               # E       RuntimeError: expected ) but found 'ident' here:
```

Comment:
could you file an issue for this and link it here? it's just easier to find these skips to be removed once the fixes are landed. In this case, I am working on the complex JIT support and https://github.com/pytorch/pytorch/pull/52881 should resolve this issue.